### PR TITLE
Fix deploy backward migrations

### PR DESCRIPTION
**Pull request by mkurek**
Fixed backward migrations for feature/deployment branch

### CODE REVIEW

File: src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py [hunk 1]
```diff
@@ -20,7 +20,12 @@ def move_from_base_object_to_ethernet(apps, schema_editor):
 
 
 def move_from_ethernet_to_base_object(apps, schema_editor):
-    pass
+    Ethernet = apps.get_model('assets', 'Ethernet')
+    for eth in Ethernet.objects.filter(
+        ipaddress__isnull=False,
```
Comment (sputnikci) [inline src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py]: \[Pylint\] INFO: Wrong hanging indentation before block \(add 4 spaces\)\.
        ipaddress\_\_isnull=False,
        ^   | \[bad\-continuation\]

File: src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py [hunk 2]
```diff
@@ -20,7 +20,12 @@ def move_from_base_object_to_ethernet(apps, schema_editor):
 
 
 def move_from_ethernet_to_base_object(apps, schema_editor):
-    pass
+    Ethernet = apps.get_model('assets', 'Ethernet')
```
Comment (sputnikci) [inline src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py]: \[Pylint\] INFO: Invalid variable name "Ethernet" \[invalid\-name\]

File: src/ralph/networks/models/networks.py [hunk 3]
```diff
@@ -26,6 +28,8 @@
 from ralph.networks.fields import IPNetwork
 from ralph.networks.models.choices import IPAddressStatus
 
+logger = logging.getLogger(__name__)
```
Comment (sputnikci) [inline src/ralph/networks/models/networks.py]: \[Pylint\] INFO: Invalid constant name "logger" \[invalid\-name\]

File: src/ralph/networks/models/networks.py [hunk 4]
```diff
@@ -585,4 +589,9 @@ def rebuild_handler(sender, **kwargs):
     """
     # post_migrate is called after each app's migrations
     if sender.name == 'ralph.' + Network._meta.app_label:
-        Network.objects.rebuild()
+        try:
+            Network.objects.rebuild()
```
Comment (sputnikci) [inline src/ralph/networks/models/networks.py]: \[Pylint\] ERROR: Class 'Network' has no 'objects' member \[no\-member\]

File: src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py (hunk 1 above)
Comment (sputnikci) [inline src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py]: \[Pylint\] INFO: Wrong hanging indentation before block \(add 4 spaces\)\.
        ipaddress\_\_isnull=False,
        ^   | \[bad\-continuation\]

File: src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py (hunk 2 above)
Comment (sputnikci) [inline src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py]: \[Pylint\] INFO: Invalid variable name "Ethernet" \[invalid\-name\]

File: src/ralph/networks/models/networks.py (hunk 3 above)
Comment (sputnikci) [inline src/ralph/networks/models/networks.py]: \[Pylint\] INFO: Invalid constant name "logger" \[invalid\-name\]

File: src/ralph/networks/models/networks.py (hunk 4 above)
Comment (sputnikci) [inline src/ralph/networks/models/networks.py]: \[Pylint\] ERROR: Class 'Network' has no 'objects' member \[no\-member\]

File: src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py (hunk 1 above)
Comment (sputnikci) [inline src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py]: \[Pylint\] INFO: Wrong hanging indentation before block \(add 4 spaces\)\.
        ipaddress\_\_isnull=False,
        ^   | \[bad\-continuation\]

File: src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py (hunk 2 above)
Comment (sputnikci) [inline src/ralph/networks/migrations/0003_custom_link_ips_to_eth.py]: \[Pylint\] INFO: Invalid variable name "Ethernet" \[invalid\-name\]

File: src/ralph/networks/models/networks.py (hunk 3 above)
Comment (sputnikci) [inline src/ralph/networks/models/networks.py]: \[Pylint\] INFO: Invalid constant name "logger" \[invalid\-name\]

File: src/ralph/networks/models/networks.py (hunk 4 above)
Comment (sputnikci) [inline src/ralph/networks/models/networks.py]: \[Pylint\] ERROR: Class 'Network' has no 'objects' member \[no\-member\]

**Comment by ar4s** [comment]
:ok:

**Pull request merged**